pro/src/pages/Offerers/Offerer/VenueV1/VenueEdition/VenueProvidersManager/StocksProviderForm/StocksProviderForm.module.scss

```
@@ -2,4 +2,9 @@
 
 .stocks-provider-form {
     margin-top: rem.torem(1em);
+}
+
+.aide-stock-button {
```

Petite suggestion pour uniformiser le css, on n'utilise pas directement les rem, et en général on utilise `rem.torem()` avec les px
```suggestion
.aide-stock-button {
    margin-top: rem.torem(32px);
    font-size: rem.torem(12px);
}
```